### ISSUE TITLE
Cannot See Atom Window with macOS 10.13.6  

### ISSUE THREAD

**Issue by elvinos** [atom/atom#17727]
### Prerequisites

* [ X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: https://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Atom will startup but fail to load any windows when trying to use the program. This issue occurs both in safe mode and normal. I have downgraded to 1.27.2 where atom works fine again. Potentially a problem with the latest electron upgrade or the fact that I am using macOS 10.13.6 as the issue was noticed around the time the OS was updated. 

### Steps to Reproduce

1. Open atom 
2. Applications boots but no windows are visible. I have tried opening files and using the command line to reset the view but to no prevail
3. Behaviour is repeated in safe mode
4. I have tried all three versions of 1.28 without any success

**Expected behavior:** [What you expect to happen]
Can see the editing screen

**Actual behavior:** [What actually happens]
App boots but no screen loads, repeated in safe mode. Unable to use software

**Reproduces how often:** [What percentage of the time does it reproduce?]
All the time, the application now unusable. 

### Versions
- 1.28.0
- 1.28.1
- 1.28.2

This is the second time I have posted this as the issue was closed with no success. Please let me know what other information I can provide to help replicate the bug.

**Comment by daviwil** [comment]
Could you give us the output of running `atom --foreground` with any of the 1.28.0 releases?

**Comment by daneuesden** [comment]
I'm having the same issue (i'm quite a novice with this stuff so sorry if it doesn't help). I've installed the shell commands, with atom open i've run the `atom --foreground` command in terminal but get no output - a new instance of atom opens in the launcher but immediately shuts - no windows visible.

**Comment by rsese** [comment]
@daneuesden - what version of Atom did you install (does `atom -v` work?) and what version of macOS are you running?

Also, did you try [safe mode](https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode) or [temporarily resetting to factory defaults](https://flight-manual.atom.io/hacking-atom/sections/debugging/#reset-to-factory-defaults)?

You mentioned that you installed the shell commands so just to clarify, Atom worked for you at some point but then stopped working?

**Comment by daneuesden** [comment]
@rsese - `atom -v` does work, output below;
```
Atom    : 1.29.0
Electron: 2.0.5
Chrome  : 61.0.3163.100
Node    : 8.9.3
```
Actually i've also noticed this is on 1.29 build whereas this issue is 1.28.

I'm running Mac OS version 13.10.6.

I've not tried safe mode or temp. resetting factory defaults but will do and get back to you.

Atom worked prior to the 13.10.6 release but since the OS upgrade it's opened but no windows show as initially reported.

**Comment by elvinos** [comment]
<img width="644" alt="screenshot" src="https://user-images.githubusercontent.com/10270600/43576316-f341092e-9640-11e8-9899-c7c7130b3dfe.png">

Atom foreground yields the same results for me as @daneuesden. An instance of atom loads but no visible window appears. This is with:
- Atom    : 1.29.0
- Electron: 2.0.5
- Chrome  : 61.0.3163.100
- Node: 8.9.3

**Comment by rsese** [comment]
It looks like slightly different in both cases though since @elvinos for you `atom --foreground` doesn't exit (based on your screenshot) but it does in @daneuesden's case?

Do either of you see any crash reports?

https://flight-manual.atom.io/hacking-atom/sections/debugging/#find-crash-logs

@elvinos in your case, is there a dock item active for Atom after you start it?  And can you <kbd>cmd-tab</kbd> to the application and see the menu bar?

**Comment by daneuesden** [comment]
@rsese I don't see any crash reports, Atom doesn't actually crash per-se. The dock icon is still active in the launcher window (and menu bar is visible) and is theoretically running with no problem but you can't see any of the windows.

Screenshot of console.app report list for reference;
![screen shot 2018-08-03 at 09 24 01](https://user-images.githubusercontent.com/14224754/43633070-0e2bdd8a-9700-11e8-89e8-97067ee2ba68.png)

**Comment by rsese** [comment]
> @rsese I don't see any crash reports, Atom doesn't actually crash per-se. The dock icon is still active in the launcher window (and menu bar is visible) and is theoretically running with no problem but you can't see any of the windows.

@daneuesden - is the menu bar completely visible with "File, Edit, View", etc.?  Can you do things like `File > New Window` and the window is listed under the `Window` menu item?

**Comment by daneuesden** [comment]
@rsese yes that's correct. I can see the full menu bar. Opening a new Windows lists it in the dock icon. Not explicitly looked in the window menu though

**Comment by rsese** [comment]
> Opening a new Windows lists it in the dock icon. 

Hmmm, similarly is there anything in the `Window` menu that would indicate that a window is open @elvinos @daneuesden?  If you click on `Window > Zoom` does that bring up any window?

**Comment by rsese** [comment]
Oh one more thing - when you start Atom, do you notice any Atom-related entries in `/var/log/system.log`?  There's one or 2 issues a while back where there actually was useful information there.

**Comment by rsese** [comment]
Ahhh great, thanks for confirming @daneuesden :+1:  Based on the description, it sounded like things were working but the windows were just not visible for some reason.  @elvinos @marco-brandizi - can you try the suggestion [above](https://github.com/atom/atom/issues/17727#issuecomment-410849914) and see if that helps?

**Comment by rsese** [comment]
That's https://github.com/atom/atom/issues/12508 @jrtibbetts but you did find the the [pull request](https://github.com/atom/atom/pull/17529) opened to resolve that issue if you want to subscribe there.

**Comment by marco-brandizi** [comment]
@rsese, nope, I initially see the main windows, but then there are a lot of menu items and other actions (About, Licence, reopen last project folder, drag&drop files from Finder) that don't yield any action.

**Comment by elvinos** [comment]
After clicking the `Zoom` button in the menu bar, the problem seems to have been resolved.

It is strange how trying just about anything else seem to have no effect in making the windows visible.

Anyway thanks Team Atom for the help!

**Comment by daviwil** [comment]
Glad that solved the issue for you!  I think we should keep this issue open though in case there's a regression in Atom causing this problem.

**Comment by rsese** [comment]
@juliauser - that sounds like a different issue than what's reported here.  Go ahead and open a new issue and fill out the issue template and we can take a look.

**Comment by jucavi** [comment]
@rsese 
> I have a similar. I am running on a Mac os 10.14.2 and I cannot open any window in Atom. The only thing that appears in the menu bar is 'Atom'. And if I select it, I only see 'Quit'.
> 
> Any help? Thanks

Same problem.

Atom    : 1.35.1
Electron: 2.0.18
Chrome  : 61.0.3163.100
Node    : 8.9.3
 
In macOS High Sierra 10.13.16

kaos:~/ $ **atom --foreground**                                                                                                      [20:15:13]
2019-03-18 20:16:42.498 Atom[1115:98628] Syntax error on line 4, column 15: Unexpected IdentifierLiteral
2019-03-18 20:16:42.499 Atom[1115:98628] SyntaxError: Syntax error on line 4, column 15: Unexpected IdentifierLiteral
    at transformNode (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:214:13)
    at Value (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:97:12)
    at transformNode (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:216:12)
    at addProperty (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:147:15)
    at Array.reduce (<anonymous>:null:null)
    at Obj (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:140:28)
    at transformNode (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:216:12)
    at Value (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:97:12)
    at transformNode (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:216:12)
    at addProperty (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:147:15)
    at Array.reduce (<anonymous>:null:null)
    at Obj (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:140:28)
    at transformNode (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:216:12)
    at Value (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:97:12)
    at transformNode (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:216:12)
    at addProperty (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:147:15)
    at Array.reduce (<anonymous>:null:null)
    at Obj (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:140:28)
    at transformNode (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:216:12)
    at Value (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:97:12)
    at transformNode (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:216:12)
    at addProperty (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:147:15)
    at Array.reduce (<anonymous>:null:null)
    at Obj (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:140:28)
    at transformNode (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:216:12)
    at Value (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:97:12)
    at transformNode (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:216:12)
    at Block (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:94:12)
    at transformNode (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:216:12)
    at Object.parse (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/cson-parser/lib/parse.js:222:16)
    at parseObject (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/season/lib/cson.js:42:23)
    at parseContentsSync (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/season/lib/cson.js:68:16)
    at Object.readFileSync (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/season/lib/cson.js:180:14)
    at getConfig (/Applications/Atom.app/Contents/Resources/app.asar/src/main-process/start.js:117:33)
    at start (/Applications/Atom.app/Contents/Resources/app.asar/src/main-process/start.js:47:18)
    at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/main-process/main.js:62:1)
    at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/main-process/main.js:64:3)
    at Module._compile (module.js:642:30)
    at Object.Module._extensions..js (module.js:653:10)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/electron.asar/browser/init.js:186:8)
    at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/electron.asar/browser/init.js:188:3)
    at Module._compile (module.js:642:30)
    at Object.Module._extensions..js (module.js:653:10)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Function.Module.runMain (module.js:683:10)
    at startup (bootstrap_node.js:196:16)
    at bootstrap_node.js:622:3

**Comment by jucavi** [comment]
![Captura de pantalla 2019-03-18 a las 20 30 37](https://user-images.githubusercontent.com/102237/54558000-d5228780-49bc-11e9-8304-d3645fdd6a59.png)

**Comment by rsese** [comment]
@jucavi - your issue looks different since you're getting an error and the problem of the initial report was resolved by zooming the window.

Also, your screenshot shows the command was run as `atom --save`, it should be `atom --safe`.  If `atom --safe` doesn't help, I'd suggest [temporarily resetting to factory defaults](https://flight-manual.atom.io/hacking-atom/sections/debugging/#reset-to-factory-defaults).

If you still have trouble after that, feel free to post on the [official message board](https://discuss.atom.io/) and @ mention me there and we can take a look together.

**Comment by claudioasfilho** [comment]
If you click Window->Zoom it comes back.

**Comment by grantkruger** [comment]
Just FYI I'm on MacOS Mojave -- 10.14.3 with 
Atom    : 1.35.1
Electron: 2.0.18
Chrome  : 61.0.3163.100
Node    : 8.9.3

I was still getting this problem. The zoom trick kinda worked, as in it opened the current project only, not other open projects and you had to do it every time to opened the project. Also, reopen project did not work at all, not even with the zoom trick. I had to open the project again and then click zoom. Reopen + zoom did nothing. However the zoom trick gave me access to preferences and updates again and after running updates of installed packages Atom seems to work fine again. If I'd known that would happen I'd have paid more attention to what packages were updated and be able to tell you...but I didn't. All I can tell you is there were 8 of them. If it happens again I'll pay more attention, but if this works for others maybe they can too.

**Comment by arunmummidi** [comment]
I faced same issue. But I was able to fix it by following below steps:

1) Go to View > Show all tabs

This will pop up all the tabs. Then you can use Atom normally.

Thanks!
Arun

**Comment by ashwanijha04** [comment]
I observed that the window size of atom had been reduced so much that it couldn't be noticeable or open.
You can go to views -> Toggle Full screen and then resize the window again.

**Comment by Fomtriok** [comment]
Thanks rsese, that was the same problem I had. Strange that the window size would be reduced so much.

**A preventive fix for Mac users is: Changing the dock settings to "Double-click a window's title bar to: Minimize" [instead of Zoom].**

For one can [when that setting is set to zoom, which I think is the default] [and when Atom is active and the window _should_ be visible], simply double click 2 % from the bottom of the screen in the far left corner, and that has the same effect as pressing "Window -> Zoom". Then double clicking on the title bar, once again reduces window size to being nearly being invisible.

I think that is actually how most people originally got this problem; that they at some point accidentally or intentionally double clicked the title bar of an Atom window. But if they had other windows active still, I think whenever they open a file in Atom, the new file might often be loaded to that visible window. Not until they - possibly days or weeks later - close that visible window, will new files opened with Atom load to that former "open" window, which is currently 99.9 % hidden in the far left bottom corner of the screen. Although not actually random, it's is perceived as random.

**Comment by har12345** [comment]
Hi, I had the same issue. The issue fixed when i changed the System Settings in Display from 'Scaled' to 'Default for this display'

MacOS: 10.14.6 Mojave
Atom    : 1.42.0
Electron: 4.2.7
Chrome  : 69.0.3497.128
Node    : 10.11.0

**Comment by sblackstone** [comment]
I've just experienced this issue on the latest version of atom (1.44.0) - Window -> Zoom fixed...

**Comment by pdlla** [comment]
same issue, same fix :)

It seems the windows settings are saved. Not even a restart could fix it.

**Comment by bl4ck5un** [comment]
same issue, same fix :)

On macOS 10.15.3 (19D76) with Atom:

```
Atom    : 1.44.0
Electron: 4.2.7
Chrome  : 69.0.3497.128
Node    : 10.11.0
```

**Comment by pinkfloyda** [comment]
Tried above, If you click Window->Zoom it comes back. it worked, actually if you look carefully in left bottom of screen, the window is minimized there, you can try manually resize.

Not sure why it suddenly becomes so small, probably a bug

**Comment by mime29** [comment]
> Tried above, If you click Window->Zoom it comes back. it worked, actually if you look carefully in left bottom of screen, the window is minimized there, you can try manually resize.
> 
> Not sure why it suddenly becomes so small, probably a bug

Same here Atom v 1.44.0 x64
Mac Os X 10.15.3 (19D76)

**Comment by dhazeghi** [comment]
Same here.

Window is shrunk to literally just a few pixels, almost off-screen, on the left side.  Perhaps related to switching between internal/external monitors?

atom 1.45.0
macOS 10.14.6

**Comment by stefanocoretta** [comment]
Same problem. Fixed with `Zoom`.

```
atom 1.45.0
macOS 10.15.3 (19D76)
```

**Comment by mgrosman** [comment]
Same here- window>Zoom fixed it for me as well. What a silly bug!

**Comment by charger71** [comment]
The same, Zoom fixed the issue for me, thanks!

**Comment by leonp** [comment]
Same issue, Window → Zoom “fixes” the problem

Started today.

- Catalina 10.15.3
- Intel Iris Graphics 6100 1536 MB
- Atom 1.45.0

**Comment by ywingdriver** [comment]
I have been struggling with atom not opening for 6 months and the zoom button fixed the problem. Thank you!

**Comment by jonathandhn** [comment]
> After clicking the `Zoom` button in the menu bar, the problem seems to have been resolved.
> 
> It is strange how trying just about anything else seem to have no effect in making the windows visible.
> 
> Anyway thanks Team Atom for the help!

Mojave 10.14.6
Intel Iris Graphics 6100 1536 MB
Atom 1.45.0

Started on the full system crash while atom being in full screen at 2nd monitor.

Ok please can someone do the thing for updating help page on atom.io, I delete every thing for nothing (applet, setup), because there are missing information about the "ZOOM" button tricks on [Debugging](https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode)

**Comment by mbauman** [comment]
This is definitely related to having multiple monitors and connecting/disconnecting them with Atom running.  Note that the window _is still there_, it's just horribly shaped.  In my particular case, it ends up being a ~10x2 pixel shape on the left side of the window:

![](https://user-images.githubusercontent.com/154641/79890041-84057400-83c4-11ea-8750-ecd8d72ee879.png)

That little smudge on the left? Yup, that's the Atom window.  It's also the _default_ size for new atom windows, so they all just pile up right there.  Here's what happens when I activate Expose:

![](https://user-images.githubusercontent.com/154641/79890447-1dcd2100-83c5-11ea-9911-478323ffdfe6.gif)

**Comment by maxzhu213** [comment]
For the record, I am using only one monitor and this just happened to me, and Zoom fixed it.
`% atom -v
Atom    : 1.47.0
Electron: 5.0.13
Chrome  : 73.0.3683.121
Node    : 12.0.0`

**Comment by glwhart** [comment]
I also have been plagued by the "invisible windows" bug. The "Zoom" trick however didn't always work for me. I noticed thanks to https://github.com/atom/atom/issues/17727#issuecomment-630958312 that there is still a small sliver of the windows visible. I was able to grab the edge and re-expand the windows. This will probably be my workaround for now.

Almost exclusively, I use my computer with an external monitor and the clamshell closed. I rarely use it with the clamshell open.

**Comment by danieldiaz-mule** [comment]
2 years later and the issue is still present... Zoom fixed it, though.

For the record:
d.diaz@danieldiaz-ltm aws % atom -v && sw_vers
Atom    : 1.49.0
Electron: 5.0.13
Chrome  : 73.0.3683.121
Node    : 12.0.0
ProductName:	Mac OS X
ProductVersion:	10.15.6
BuildVersion:	19G73

**Comment by clean-code-charlie** [comment]
My case: it has happened before on windows, today is the first time it happens on mac (mini 2014 Mojave).
What I tried: quit, reopen, hide/show window, open a new window, open a file, create new file, go to settings... nada, problem remains
This solved the issue: from the file menu, open a file in dev mode, window is still not showing up so I just quit then reopen and the new window appeared!

Hope it helps.

**Comment by steroman** [comment]
@clean-code-charlie thanks, your workaround worked for me on MacBook Pro 2012 on Mojave 10.14.6

**Comment by kent** [comment]
My case: Turns out that when I doubled click the Window's header bar, it minimized the window down into a 10x10px in the bottom 0,0 of my screen (left, bottom). I have a dark background and didn't see it. It was also in some kind of Full Screen mode, but wasn't full screen.
I dragged to resize the window. 
Closed it with the red x.
Closed Atom.
Re-opened and was able to reproduce with `-v 1.53.0`.
Applied the same fix, and all is fine now, but I can't double click on the window bar. 
I ended up using a Window manager from Setapp called Mosaic to fix the window problem

**Comment by solex16** [comment]
I've been using Atom for 4 years and had this problem..

Window -> Zoom fixed it..

I feel stupid, I was using Visual Studio for a few days with none of my confgis set up, 
and nearly started using it until I found this thread..

Seems like a trivial problem with big consequences that could possibly easily be fixed?..

**Comment by amk221** [comment]
For me this happened because I had symlinked `~/.atom/` to a location, which I then moved. I just had to recreate the link to fix it.

**Comment by wadewilliams** [comment]
Lost enough time on this between yesterday and today I almost stopped using atom. Window > Zoom works, but this is insane.

**Comment by TomHall2020** [comment]
This caught me out today, think I did double click the top bar, saw the animation for the window zooming down to the bottom left corner but didn't realise what it was at the time. Problem persisted even after restarting the app and the whole machine. hitting zoom fixed it. Perhaps change the default zoom down setting to just half size the screen instead of shrinking it to nothing?

Mac Os 10.15.4
Atom    : 1.53.0
Electron: 6.1.12
Chrome  : 76.0.3809.146
Node    : 12.4.0

**Comment by old-square-eyes** [comment]
MacOS - 10.15.7

Atom    : 1.53.0
Electron: 6.1.12
Chrome  : 76.0.3809.146
Node    : 12.4.0

Clicking around a lot and window zoom brings it back. But it's slow and disruptive to recover.

**Comment by AlbertKang1004** [comment]
For me, the Zoom option is disabled... I am using macOS 11.0 Beta.
Atom : 1.53.0
Chrome : 87.0.4280.88 
<img width="307" alt="Screen Shot 2020-12-19 at 7 58 06 PM" src="https://user-images.githubusercontent.com/64033677/102702780-84485e80-4234-11eb-96d2-88ba9f3b1dde.png">

**Comment by rmurrant** [comment]
Most of the options on here didn't show for me, but after a lot of trying, one of the suggestions pointed me to the right answer.  I think under View (can't remember now), there is an option to toggle developer mode.  After doing that, quit Atom, and reopen Atom.  Fixed!

**Comment by carinemores** [comment]
The only thing that worked for me was starting atom with `atom --clear-window-state`

**Comment by JonKernPA** [comment]
I eventually **succeeded**. Read on.

Versions:

Atom    : 1.54.0
Electron: 6.1.12
Chrome  : 76.0.3809.146
Node    : 12.4.0
Mac OS X: 10.15.7 (19H2)

-----

So I installed Atom as follows:

- Download (clicked link on home page and it knew it was for Mac OS X)
- Had Mac open download with archive utility
- Dragged the Atom file to the Applications folder
- Started up Atom (once with Find, and once with app launcher)
- Atom was basically dead... no window, no menus with Command Pallette. Nothing
- Tried it twice.
- New terminal window revealed nothing when testing
```
$ which atom
$ atom
-bash: atom: command not found
```
- Quit Atom

**SUCCESS**: Then I went to Finder and to the Applications folder.
- I double-clicked the Atom file
- Atom opened in all of its glory.

```
$ which atom
/usr/local/bin/atom
```

**Comment by cleveranecdote** [comment]
Just happened to me as well. Dual monitor but was in single monitor and rebooted.

Catalina 10.15.7
Atom 1.54.0

Zoom worked.

**Comment by Thomas-1985** [comment]
Same here

Catalina 10.15.7
Atom 1.54.0

**Comment by anilsezar** [comment]
Had the same error for a long time. Checked this issue, tried zoom and it worked. 

Atom: 1.55.0
Catalina 10.15.7

**Comment by gurdeeps** [comment]
Seeing the same issue on Catalina 10.15.7. Zoom does fix it. It would be nice to not have to crawl google and read through all the comments on this issue to get a workaround. I almost stopped using Atom because of this issue.

**Comment by M09NST3R** [comment]
> Seeing the same issue on Catalina 10.15.7. Zoom does fix it. It would be nice to not have to crawl google and read through all the comments on this issue to get a workaround. I almost stopped using Atom because of this issue.

Same. Was very close to dumping it. Very frustrating.

**Comment by g60madman** [comment]
Tested on BigSur 11.2.3 and Atom 1.56 x64. Thank goodness for the Zoom that fixed it so I can see the windows again but was freaking out when windows disappeared. Be nice to get this fixed.

**Comment by ericbendall** [comment]
> Seeing the same issue on Catalina 10.15.7. Zoom does fix it. It would be nice to not have to crawl google and read through all the comments on this issue to get a workaround. I almost stopped using Atom because of this issue.

Same, went a couple of days before seeing this thread. `Zoom` worked. I was a few minutes away from switching editors. I think this is a UX defect, though I'm not sure what exactly the solution is.

**Comment by g60madman** [comment]
> > Seeing the same issue on Catalina 10.15.7. Zoom does fix it. It would be nice to not have to crawl google and read through all the comments on this issue to get a workaround. I almost stopped using Atom because of this issue.
> 
> Same, went a couple of days before seeing this thread. `Zoom` worked. I was a few minutes away from switching editors. I think this is a UX defect, though I'm not sure what exactly the solution is.

Not sure what is going, as it stands I am getting tired of OS X been having too many issues with Bluetooth with my MX keyboard/mouse and AirPods. It's exhausting...I even switched to the darn dongle and still having issues.

**Comment by old-square-eyes** [comment]
Slightly off topic but wanting to leave here since it's startup related. I noticed recently opening and closing a lot of text files, that Atom takes way too long to start. A text editor should be light. With a performant M1 Mac, and SSD etc I'd expect Atom to be quite snappy, currently ~7 full seconds from double clicking a small text file to seeing the contents. Are there things that can be loaded after the fact so that the initial file open doesn't have to take so long? Doesn't seem to be a problem with default text editors or Sublime, for example... which open files almost instantaneously.

**Comment by TonyzBi** [comment]
> Same issue, Window → Zoom “fixes” the problem
> 
> Started today.
> 
> * Catalina 10.15.3
> * Intel Iris Graphics 6100 1536 MB
> * Atom 1.45.0

It is OK for my Mac, Big Sur 11.5.1

**Comment by m-an** [comment]
Still an issue. Just faced it today and found that little line that was zoomed out Atom window thanks to this thread

**Comment by z0r3f** [comment]
Still occurring:
```zsh
~ % atom -v
Atom    : 1.58.0
Electron: 9.4.4
Chrome  : 83.0.4103.122
Node    : 12.14.1
```

Not update available.
Zoom fixed.

**Comment by dp181221** [comment]
![Screenshot 2022-01-05 at 20 13 26](https://user-images.githubusercontent.com/95494525/148312801-f000c88a-649b-4e2b-bfc0-3c144d479a49.png)
Still the same issue? zoom option not available. is there any other fix to this issue?

**Comment by marcomoscatiello** [comment]
i keep seeing this issue in Atom (1.58) and Macos Monterey 12.1 .
I struggle to show the windows and if i close and reopen Atom , everything is still hidden and the only way to fix it is either use Window -> Zoom and close the windows one by one or Force Quit through MacOs. 
Is there any issue open on this yet?
Thank you

**Comment by ahaitoute** [comment]
Same issue with Mac OS Catalina and atom 1.59.0.
I see on the left bottom of the screen a barely visible lines, when hovering over one of the lines, I can resize this line to a bigger window.

![Atom issue 17727-1](https://user-images.githubusercontent.com/3607632/154010584-00d315a6-1bd7-4fe3-ae0b-7dd7ccdbaf3b.png)
![Atom issue 17727-2](https://user-images.githubusercontent.com/3607632/154010616-70a35c30-0356-4dbe-84c7-83717bfe5645.png)

**Comment by a2isystems-vip** [comment]
I see the same issue on Mac Monterey

**Comment by AlbertSnows** [comment]
> ![Screenshot 2022-01-05 at 20 13 26](https://user-images.githubusercontent.com/95494525/148312801-f000c88a-649b-4e2b-bfc0-3c144d479a49.png) Still the same issue? zoom option not available. is there any other fix to this issue?

also seeing this on Montery.

**Comment by pasindujw** [comment]
The 'zoom' button saved my day. 
If anyone else observing this in Mac, please try that.
Kudos to one who mentioned that here.

**Comment by William83968** [comment]
Just in case if anybody is stuck on this problem as well.
My problem was that Atom window wasn't showing as well and the zoom option was disabled.
I was so frustrated that I couldn't find any other solutions until from the discussions above I realised I haven't tried all the solutions yet.
What I did was I found out that somehow Atom was assigned to None which was probably the reason why it wasn't showing up.
To tryout my solution(I high suggest this for anybody who is still stuck on this problem):
1. Right click on the app icon
2. Click on 'Options'
3. Make sure it's not assigned to None if so, then changed it
4. Use rm -rf ~/.atom sudo open -a Atom command to open it again
It solved my problem and Atom was working correctly.
Anyway, this solution will probably work. Another thing that you can probably try was 'toggle developer tools', which might help.
